src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx

```diff
@@ -29,102 +30,143 @@ const UserMenu: FC<UserMenuProps> = ({
   const { formatMessage } = useIntl();
   const { connectWallet } = useAppContext();
   const { name, profile } = user || {};
+  const [activeSubmenu, setActiveSubmenu] = useState<string | null>(null);
```

Comment:
isActiveSubmenu

Reply:
I'm setting subenu ID here and it's not boolean so I think isActiveSubmenu name doesn't fit